docs/platforms/javascript/guides/nuxt/install/cli-import.mdx

```diff
@@ -57,13 +42,13 @@ To make things easier, add a script in the `package.json`:
 ```json {filename:package.json}
 {
   "scripts": {
-    "preview": "NODE_OPTIONS='--import .//server/sentry.server.config.mjs' nuxt preview",
+    "preview": "NODE_OPTIONS='--import ./server/sentry.server.config.mjs' nuxt preview",
```

Comment:
```suggestion
    "preview": "NODE_OPTIONS='--import ./output/server/sentry.server.config.mjs' nuxt preview",
```

Reply:
I have to delete this line anyway. When running Nuxt with `nuxt preview`, the runner uses two different reference paths (one in `root/.output/` and one in `root/`). So the command will fail at some time.